scripts/lib/install

```diff
@@ -103,6 +103,12 @@ fi
 if [ "$has_appserver" = 0 ]; then
     "$ZULIP_PATH"/scripts/setup/generate_secrets.py --production
     cp -a "$ZULIP_PATH"/zproject/prod_settings_template.py /etc/zulip/settings.py
+    if [ -z "$EXTERNAL_HOST"]; then
+        echo -e "\nEXTERNAL_HOST = $EXTERNAL_HOST" >> /etc/zulip/settings.py
+    fi
+    if [ -z "EMAIL_HOST"]; then
+        echo -e "\nEMAIL_HOST = $EMAIL_HOST" >> /etc/zulip/settings.py
+    fi
```

Comment:
Replace these with using `sed` to modify the line present in `prod_settings_template.py`.  We don't want 2 conflicting lines in the file, it'll confuse users who try to modify things manually.

Also, I don't think `EMAIL_HOST` should be one of the settings we support here.  I think we want `EXTERNAL_HOST` and `ZULIP_ADMINISTRATOR` to start.